### PR TITLE
Use a variable for foreground/text color

### LINKED ISSUE #121: Changing tooltip text color with variables
Would like to be able to change not only the background color but also the foreground color of the tooltips using available css variables like what we already have.

### PR DESCRIPTION
**Pull request by moeenio**
fix #121

### CODE REVIEW

File: src/balloon.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 :root {
   --balloon-color: #{$balloon-bg};
+  --balloon-foreground-color: #fff;
```
Comment (kazzkiq) [inline src/balloon.scss]: For ease of use, can you please just swich the name to `--balloon-text-color`?

```suggestion
  --balloon-text-color: #fff;
```

**Comment by kazzkiq** [comment]
Thanks for the addition!

**Comment by moeenio** [comment]
You're welcome, thanks for merging!

**Comment by Kristinita** [comment]
**Type: Suggestion** ➕

Please, add information about this option to:

1. [**Official Balloon.css description page**](https://kazzkiq.github.io/balloon.css/)
1. [**Balloon.css GitHub README**](https://github.com/kazzkiq/balloon.css#customizing-tooltips)

Currently, it’s not easy for users to know that this option exists.

Thanks.

**Pull request merged**